frontend/src/context/lang.ts

```diff
@@ -0,0 +1,3 @@
+import { createContext } from 'react';
+
+export const LangContext = createContext([] as  any);
```

Comment:
Could we type this? `createContext<[SupportedLocale, func] | []>([])`  (I'm not sure if that's exactly right...)

Reply:
Will try that out